packages/mui-material/src/generateUtilityClass/index.ts

```diff
@@ -0,0 +1,2 @@
+export { unstable_generateUtilityClass as default } from '@mui/utils';
+export type { GlobalStateSlot } from '@mui/utils';
```

Comment:
Why do you reexport these? Shouldn't all the components use class name utility from `@mui/utils`?